driver/razeraccessory_driver.h

```diff
@@ -9,6 +9,7 @@
 #define USB_DEVICE_ID_RAZER_FIREFLY_HYPERFLUX 0x0068
 #define USB_DEVICE_ID_RAZER_MOUSE_DOCK 0x007E
 #define USB_DEVICE_ID_RAZER_CORE 0x0215
+#define USB_DEVICE_ID_RAZER_CORE_X_CHROMA 0x0F1A
```

Comment:
Just move this down below USB_DEVICE_ID_RAZER_KRAKEN_KITTY_EDITION to keep sorted